reconcile/jira_permissions_validator.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import sys
+
+from reconcile import queries
+from reconcile.status import ExitCodes
+from reconcile.utils.jira_client import JiraClient
+
+QONTRACT_INTEGRATION = "jira-permissions-validator"
+
+
+def run(dry_run: bool) -> None:
+    settings = queries.get_app_interface_settings()
```

Comment:
Since this is a new integration, can we use typed queries instead of queries.py?